### DISCUSSION TITLE
Custom Fields in Single-Issue-View

### DISCUSSION THREAD

**Discussion by theSmokingFrog** [ankitpokhrel/jira-cli#293] [Q&A] [answered]
When using `jira issue view <ISSUE>` standard fields of Jira are shown in a readable manner.

Now let's talk about custom fields:
The project I'm working with actively uses an agile Workflow, thus our Backlog Items have a custom field called "Acceptance Criteria" as part of the Jira Issue.

- Is it possible to display custom fields with the `view` command?
- is it possible to define the order in which these fields are displayed?

If thats not the case - I would love to see this as a feature request.

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @theSmokingFrog, unfortunately this is not possible at the moment. I don't have any immediate plan to address this to keep things simple. We can come to this later if there is enough usecase/request for this feature.

**Reply by theSmokingFrog** [reply]
Thanks for the quick reply. I thought so as I was not finding settings related to this.
I certainly hope that this will become a thing in the future as this is already a valuable tool.

**Comment by php-coder** [comment]
Yes, it would be great as it helps in scripting. For example, I'd like to get issue's summary only. In order to make it generic across all commands (`issue`, `sprint`, etc), I'd suggest to introduce `--output <format>` option where "format" can be json, yaml, plain, etc (like it's implemented in `kubectl`). In this case, uses will be able to use external tools (like `jq`) in order to get results that we need.

**Reply by smahm006** [reply]
I mean even something as simple as "I want to see which Epic this issue belongs to" is not possible right now with view.

Another [Jira CLI project](https://github.com/amritghimire/jira-terminal) I noticed does include this feature.
